README.md

```diff
@@ -442,29 +442,82 @@ module.exports = function(app) {
   });
 };
 ```
+#### Migrating models with GeoPoint types before loopback-connector-mysql@5.x
+Previous to `loopback-connector-mysql@5.x`, MySQL connector was saving and loading GeoPoint properties from the MySQL database in reverse.
+MySQL expects values to be POINT(X, Y) or POINT(lon, lat), but the connector was doing the opposite. If you have an application with a model
+that has a GeoPoint property using previous versions of this connector, you can migrate your models using the following programmatic approach:
+**NOTE** Please back up the database tables that have your application data before performing any of the steps.
+1. Create a boot script under `server/boot/` directory with the following:
```

Comment:
Good catch both :). I've added a second step, but can remove it if it's too trivial.